linode/manager
# DBaaS1-78: Landing page empty state

## Description
Update the DBaaS landing page empty state copy and links, as well as a small update to the Create dropdown description for Databases.

## How to test
With the MSW on, visit `/databases` to see the empty state. The entry for Databases in the Create menu dropdown should also be updated to read, `High-performance managed database clusters`.

<img width="899" alt="Screen Shot 2022-01-18 at 2 58 45 PM" src="https://user-images.githubusercontent.com/32860776/150010432-9f475652-04e0-441c-9fd5-fb314310a836.png">

<img width="368" alt="Screen Shot 2022-01-18 at 3 05 56 PM" src="https://user-images.githubusercontent.com/32860776/150010543-64126dbe-5387-423b-9e05-3125a01c5545.png">

## Note to Reviewers
The Docs link used (`https://www.linode.com/docs/products/database`) is what they've penciled in as what the link will actually be. Let's use it for now and if it changes in the next week or two we can update it accordingly.
